tzconvert.py

```diff
@@ -54,9 +65,9 @@ def adjust_datetime(naive_date, city_name, debug=False):
         naive_datetime = current_datetime
         adjusted_datetime = adjust_datetime(naive_datetime, city)
 
-        if not adjusted_datetime:
-            print("Could not fetch adjusted date time")
-            continue
+    latitude, longitude = 22.5650515, 88.391722
+    nearest_city_name = tzhelpers.get_nearest_city(latitude, longitude)
+    approximate_datetime = approx_datetime(current_datetime, latitude, longitude)
 
-        print("Naive date : %s" % current_datetime.strftime(tzhelpers.DATE_TIME_FORMAT))
-        print("Adjusted date : %s\n" % adjusted_datetime.strftime(tzhelpers.DATE_TIME_FORMAT))
+    print("\nCity nearest to {}, {}: {}".format(latitude, longitude, nearest_city_name))
+    print("\nApproximate datetime of {}, {}: {}".format(latitude, longitude, approximate_datetime))
```

Comment:
use datetime.strftime() along with tzhelpers.DATE_TIME_FORMAT for consistent output.

Reply:
Okay.